package.json

```diff
@@ -35,24 +36,23 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.18.0",
     "eslint-plugin-react-hooks": "^4.0.0",
+    "husky": "^7.0.0",
     "jest": "^26.6.3",
-    "lerna": "^3.20.1",
+    "lerna": "^5.1.0",
     "lerna-update-wizard": "^0.17.8",
     "mkdirp-promise": "^5.0.1",
     "prettier": "^2.0.0",
     "prettylint": "^1.0.0",
     "rimraf": "^3.0.0",
     "ts-jest": "^26.5.1",
     "ts-node": "^9.0.0",
-    "typescript": "^4.0.0",
-    "husky": "^7.0.0"
+    "typescript": "^4.0.0"
   },
   "scripts": {
     "bootstrap": "lerna bootstrap && lerna run build",
     "build:package": "yarn lerna run build",
     "build:sort": "lerna run build --sort",
     "build:storybook": "cd packages/storybook && yarn build:storybook",
-    "build:slow": "lerna run build",
```

Comment:
This script is still useful on lower powered machines where other cores are busy running OS / IDE etc, would suggest leaving it in

Reply:
@Moza805 I've had to use the NX commands directly (rather than using Lerna commands that NX interprets) as they give a much greater degree of flexibility, e.g. we can control the degree of parallelism with --parallel=1 to prevent any parallelism. This has meant I can add the build:slow back in. By default NX operates with a parallelism of 3.

Reply:
Thanks @paulcustance-al